image/shared/interfaces.py

```diff
@@ -191,7 +191,7 @@ def convert_manifest(
 
 
 @add_metaclass(ABCMeta)
-class ManifestListInterface(object):
+class ManifestListInterface(ManifestInterface):
```

Comment:
Classes inheriting from `ManifestListInterface` would implement the whole of `ManifestInterface` + the manifest list specific bits. Inheriting from `ManifestInterface` here just makes it official.